site/app/templates/grading/electronic/Status.twig

```diff
@@ -167,10 +167,10 @@
     {% else %}
         <div class = "tab">
             <div class =tab-bar-wrapper">
-                <a class="nav-bar normal-btn"  onclick="openStat(event, 'numerical-data'), this.blur()" id="defaultOpen">Data</a>
-                <a class="nav-bar normal-btn"  id="btn-hist-auto" onclick="openStat(event, 'autograding-score-histogram'), this.blur()">Autograding Histogram</a>
-                <a class="nav-bar normal-btn"  id="btn-comp" onclick="openStat(event, 'component-averages'), this.blur()">Manual Grading Component Averages</a>
-                <a class="nav-bar normal-btn"  id="btn-hist-tot" onclick="openStat(event, 'total-score-histogram'), this.blur()">Overall Histogram</a>
+                <a class="nav-bar normal-btn key_to_click" tabindex="0"  onclick="openStat(event, 'numerical-data'), this.blur()" id="defaultOpen">Data</a>
```

Comment:
On Firefox, this properly opens the tab but it doesn't re-highlight it as it does when clicked